samtranslator/schema/schema.json

```diff
@@ -188427,6 +188427,10 @@
           "title": "AuthorizationScopes",
           "type": "array"
         },
+        "DisableDefaultFunctionPermissions": {
```

Comment:
On second thought, how about `DisableFunctionDefaultPermissions`? To prevent confusion with the non-existent "default function".

Reply:
Had a chat with Evan about the name. He agrees that it should be `FunctionDefault` rather than `DefaultFunction` but would also prefer if we didn't use `Disable` in the name (we're trying to move away from that word). 

Some suggestions he had was `BlockDefaultAuthPermissions: True` or `NoDefaultAuthPermissions: True`. Although I think keeping `Disable` will keep the property consistent with some other SAM properties and be more familiar to customers. Thoughts on this?

Reply:
Talked offline, we decided to go with `DisableFunctionDefaultPermissions` to keep consistency and familiarity